connectors/D7SMS/apiProperties.json

```diff
@@ -0,0 +1,34 @@
+{
+  "properties": {
+    "capabilities": [],
```

Comment:
Can you please update this to : "capabilities": [ "actions" ],

Reply:
Done